web-components/src/components/alert/tokens/lm-alert-tokens.js

```diff
@@ -41,6 +41,10 @@ const alert = {
     size: {
       light: "1rem",
       dark: "1rem"
+    },
+    "text-color": {
+      light: colors.gray[90].name,
```

Comment:
Is this component called "Floating Notifications" in Figma? I see gray-100 for the light text color on that page, but maybe I'm in the wrong place

Reply:
Fixed